src/QuizProvider.tsx

```diff
@@ -3,6 +3,7 @@ import { quizReducer, initialQuizState } from "./quizReducer";
 
 const QuizContext = createContext<any>({});
 
+
```

Comment:
In line 10 and 11 there are two ways you have written the useState. You can use the first one as typescript implicitly understands it